docker_plugin/tasks.py

```diff
@@ -115,8 +105,9 @@ def start(retry_interval, daemon_client=None, **_):
 
     ctx.logger.info('Container started: {}.'.format(response))
 
-    if ctx.node.properties[('params')].get('processes_to_wait_for', False):
-        utils.wait_for_processes(retry_interval, client, ctx=ctx)
+    if params.get('processes_to_wait_for', False):
```

Comment:
Right, should be None or something. It's a leftover from when the property was boolean.